src/main/java/com/entropyteam/entropay/employees/services/ClientService.java

```diff
@@ -43,4 +49,23 @@ protected Client toEntity(ClientDto entity) {
         client.setCompany(company);
         return client;
     }
+
+    @Override
+    @Transactional
+    public ClientDto update(UUID clientId, ClientDto clientDto) {
+        Client entityToUpdate = toEntity(clientDto);
+        entityToUpdate.setId(clientId);
+        deactivateClient(clientId, entityToUpdate);
+        Client savedEntity = getRepository().save(entityToUpdate);
+        return toDTO(savedEntity);
+    }
+
+    public void deactivateClient(UUID clientId, Client clientToUpdate) {
```

Comment:
minor, i would change this method name, because is more like a validation or a check